dl/Util.py

```diff
@@ -227,6 +228,93 @@ def def_token(tok):
             return ANON_TOKEN
 
 
+def parse_auth_token(token):
```

Comment:
If this returned a dict it might be easier to use. I think it might result in more code but would make it more explicit.
```
parts = parse_auth_token(token)
```
In the above example, usage would become `parts.get('username')` instead of `parts[0]`

Reply:
Just providing my thoughts, this can be left as is or changed at your discretion :D

Reply:
That's a very good idea if these functions were oriented for users but really they are just Internal consumption, The fact that they are here, in the DL Util package, is just convenient. Also if you look at how it is used in the code, basically a predefined array, changing the return signature would end up with more code changes for no much gain in my view.